internal/ui/select.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"runtime"
 
-	"golang.org/x/crypto/ssh/terminal"
+	"golang.org/x/term"
```

Comment:
I don't know how this came through, but my linter was yelling about it so I updated it here